client/state/selectors/is-requesting-reader-teams.js

```diff
@@ -0,0 +1,3 @@
+export default function isRequestingReaderTeams( state ) {
+	return !! state.reader.teams.isRequesting;
```

Comment:
The problem with that approach is that we want to refetch that data on occasion. So we'll have a case where we both have items and a request is ongoing. There's no UI indicator for that, but we still do want to prevent multiple inflight requests from slamming the API server.